src/txs/gov/VoteForm.tsx

```diff
@@ -56,6 +58,7 @@ const VoteForm = () => {
     estimationTxValues,
     createTx,
     chain,
+    token: baseAsset,
```

Comment:
Double check vote form `Fee` value.  On branch:

<img width="838" alt="Screenshot 2023-08-18 at 10 37 06 AM" src="https://github.com/terra-money/station/assets/30275692/d9e52c9e-f4b5-41f2-9e88-fb55a973f533">


In production:

<img width="813" alt="Screenshot 2023-08-18 at 10 37 35 AM" src="https://github.com/terra-money/station/assets/30275692/4faf00c2-34c8-4ca1-a6be-a87b0d46b1d8">

Reply:
these are incorrect on prod currently and correct here